content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -397,71 +295,94 @@ The scheduler ensures that the sum of the resource requests of the scheduled Con
-basis by the kubelet.  The monitoring is performed by scanning each
-emptyDir volume, log directories, and writable layers on a periodic
-basis.  Starting with Kubernetes 1.15, emptyDir volumes (but not log
-directories or writable layers) may, at the cluster operator's option,
-be managed by use of [project
-quotas](http://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html).
-Project quotas were originally implemented in XFS, and have more
-recently been ported to ext4fs.  Project quotas can be used for both
-monitoring and enforcement; as of Kubernetes 1.16, they are available
-as alpha functionality for monitoring only.
 
-Quotas are faster and more accurate than directory scanning.  When a
-directory is assigned to a project, all files created under a
-directory are created in that project, and the kernel merely has to
-keep track of how many blocks are in use by files in that project.  If
-a file is created and deleted, but with an open file descriptor, it
-continues to consume space.  This space will be tracked by the quota,
-but will not be seen by a directory scan.
+### Monitoring ephemeral-storage consumption {#resource-emphemeralstorage-consumption}
 
-Kubernetes uses project IDs starting from 1048576.  The IDs in use are
-registered in `/etc/projects` and `/etc/projid`.  If project IDs in
-this range are used for other purposes on the system, those project
-IDs must be registered in `/etc/projects` and `/etc/projid` to prevent
-Kubernetes from using them.
+If the kubelet is managing local ephemeral storage, then the kubelet
+measures storage use in:
+- emptyDir volumes
+- log directories
+- writeable container layers
 
-To enable use of project quotas, the cluster operator must do the
-following:
+This information can trigger Pod eviction if a Pod is using more
+ephemeral-storage than you allow it.
 
-* Enable the `LocalStorageCapacityIsolationFSQuotaMonitoring=true`
-  feature gate in the kubelet configuration.  This defaults to `false`
-  in Kubernetes 1.16, so must be explicitly set to `true`.
+The kubelet supports different ways to measure Pod storage use:
+
+{{< tabs name="resource-emphemeralstorage-measurement" >}}
+{{% tab name="Periodic scanning" %}}
+The kubelet performs regular, schedules checks that scan each
+emptyDir volume, each container log directory, and each writeable container layer.
+
+The scans provide a measurement of how much space is used.
 
-* Ensure that the root partition (or optional runtime partition) is
-  built with project quotas enabled.  All XFS filesystems support
-  project quotas, but ext4 filesystems must be built specially.
+{{< note >}}
+In this mode, the kubelet does not track open file descriptors
+for deleted files. If you (or a container) create a file inside an empyDir
+volume, something then opens that file, and you delete the file whilst
+it is still open, then inode for the deleted file stays until you close
```

Comment:
```suggestion
it is still open, then the node for the deleted file stays until you close
```

Reply:
[inodes](https://en.wikipedia.org/wiki/Inode) and [nodes](https://kubernetes.io/docs/concepts/architecture/nodes/) are different things, so “node” is wrong here.